spec/unit/cli_spec.rb

```diff
@@ -200,6 +200,19 @@
         run_command(["inspect", "--skip-files=/foo/bar,/baz", example_host])
       end
 
+      it "forwards the global --exclude option to the InspectTask" do
+        expect_any_instance_of(InspectTask).to receive(:inspect_system) do |_instance, _store,
```

Comment:
Block argument expression is not on the same line as the block start.<br>Extra space before block parameter detected.